holidays/utils.py

```diff
@@ -24,6 +24,15 @@
 import holidays.financial
 from holidays.holiday_base import HolidayBase
 
+import i18n
```

Comment:
I'm wondering what's the rationale behind using `i18n` instead of `gettext` mechanism?
I strongly recommend to [re-]consider this option instead of `i18n` module and adding an extra dependency.